packages/react-native-bpk-component-text/stories.js

```diff
@@ -33,91 +33,93 @@ import CenterDecorator from '../../storybook/CenterDecorator';
 
 import BpkText, { WEIGHT_STYLES } from './index';
 
+const text = 'Lorem ipsum';
```

Comment:
This should be in UPPER_SNAKE_CASE to indicate the constant nature of this value.